plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -191,6 +191,18 @@ protected function get_settings_for_default_section() {
 					'desc_tip' => true,
 				),
 
+                array(
+					'title'    => __( 'Footer text color', 'woocommerce' ),
+					/* translators: %s: footer default color */
+					'desc'     => sprintf( __( 'The footer text color. Default %s.', 'woocommerce' ), '<code>#3c3c3c</code>' ),
+					'id'       => 'woocommerce_email_footer_text_color',
```

Comment:
Unit tests are failing due to this new setting. We'll need to update those tests to account for this: https://github.com/woocommerce/woocommerce/actions/runs/9980468073/job/27582309555?pr=49648#step:7:121